documents/src/pages/elements/interactive-chart.md

```diff
@@ -74,6 +74,8 @@ ef-interactive-chart {
 ```
 ::
 
+!> EFv6 uses lightweight chart v3.x. Upgrade to EFv7 for lightweight chart v4.x.
```

Comment:
Note that with this change, we need to deploy https://github.com/Refinitiv/refinitiv-ui/pull/857 first.
```suggestion
!> EFv6 uses lightweight-charts v3. [Upgrade to EFv7](http://ui.refinitiv.com/start/upgrade-guide) for lightweight-charts v4.
```